crypto/hash.go

```diff
@@ -108,6 +113,32 @@ func HashArgs(hash h.Hash, args ...encoding.BinaryMarshaler) ([]byte, error) {
 }
 
 // HashArgsSuite makes a new hash from the suite and calls HashArgs
-func HashArgsSuite(suite abstract.Suite, args ...encoding.BinaryMarshaler) ([]byte, error) {
+func HashArgsSuite(suite abstract.Suite, args ...interface{}) ([]byte, error) {
 	return HashArgs(suite.Hash(), args...)
 }
+
+// ConvertToBinaryMarshaler takes a slice of interfaces and returns
+// a slice of BinaryMarshalers.
+func ConvertToBinaryMarshaler(args ...interface{}) ([]encoding.BinaryMarshaler, error) {
```

Comment:
Wondering if we should let that exported or not. I don't see any reasons why it should be exported for the moment.